### PR TITLE
Allow for nested packages (package names with a /)

### PR DESCRIPTION
**Pull request by justinfagnani**
Fixes https://github.com/Polymer/polyserve/issues/204

 - [x] CHANGELOG.md has been updated

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,5 +2,6 @@ language: node_js
 node_js:
   - "node"
   - "6"
+before_install: if [[ `npm -v` != 5* ]]; then npm i -g npm@5; fi
```
Comment (aomarks) [inline .travis.yml]: this will be weird when npm 6 comes out. can you just always install npm latest instead?

File: src/make_app.ts [hunk 2]
```diff
@@ -102,3 +103,12 @@ export function makeApp(options: AppOptions): PolyserveApplication {
   app.packageName = packageName;
   return app;
 }
+
+function arrayStartsWith(path: any[], prefix: any[]) {
```
Comment (aomarks) [inline src/make_app.ts]: s/path/arr ?

File: .travis.yml (hunk 1 above)
Comment (justinfagnani) [inline .travis.yml]: true, and yes

File: src/make_app.ts (hunk 2 above)
Comment (justinfagnani) [inline src/make_app.ts]: done

**Pull request merged**